### PR TITLE
Address errors seen with Server RAID Array integrity sensor

### PR DESCRIPTION
**Pull request by shindeam**
## Problem Statement
<pre>
  <code>
    Story Ref (if any): EOS-13178
    Address errors seen with Server RAID Array integrity sensor
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Address errors seen with Server RAID Array integrity sensor
  </code>
</pre>
## Solution
<pre>
  <code>
    Change in RAIDIntegritySensor.
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [1] Yes
    - [ ] No
  </code>
</pre>
## Unit/Manual Testing Description
<pre>
  <code>
  /opt/seagate/cortx/sspl/sspl_test/run_tests.sh
  </code>
</pre>

### CODE REVIEW

**Review by thavanathan — APPROVED** [review]

File: low-level/sensors/impl/generic/raid_integrity_data.py [hunk 1]
```diff
@@ -106,10 +110,16 @@ def read_data(self):
 
     def run(self):
         """Run the sensor on its own thread"""
-
+        DEFAULT_POLLING_INTERVAL = "604800"
```
Comment (saurabhkhanvilkar) [inline low-level/sensors/impl/generic/raid_integrity_data.py:113]: can you please add a comment mentioning what this number means? microseconds/seconds/etc...

File: low-level/framework/base/sspl_constants.py [hunk 2]
```diff
@@ -143,7 +143,8 @@ class RaidDataConfig(Enum):
     MISMATCH_COUNT_RESPONSE = '0'
     RAID_RESULT_DIR = "/tmp"
     RAID_RESULT_FILE_PATH = "/tmp/result_raid_health_file"
-    MAX_RETRIES = 50
+    MAX_RETRIES = 10
```
Comment (saurabhkhanvilkar) [inline low-level/framework/base/sspl_constants.py:146]: do we need 10 retries? is 5 or even 3 sufficient?

File: low-level/sensors/impl/generic/raid_integrity_data.py (hunk 1 above)
Comment (shindeam) [reply low-level/sensors/impl/generic/raid_integrity_data.py:113]: DEFAULT_POLLING_INTERVAL has been set as 604800 seconds.
As we need to check once a week, the integrity of raid data

File: low-level/framework/base/sspl_constants.py (hunk 2 above)
Comment (shindeam) [reply low-level/framework/base/sspl_constants.py:146]: As per the observation, the RAID device file takes few seconds to change its state.
Keeping 3 or 5 retries would be less.

File: low-level/sensors/impl/generic/raid_integrity_data.py (hunk 1 above)
Comment (saurabhkhanvilkar) [reply low-level/sensors/impl/generic/raid_integrity_data.py:113]: ok so the number is in seconds... i am requesting to add a comment in code to indicate that this number is in seconds.

Comment (shindeam) [reply low-level/sensors/impl/generic/raid_integrity_data.py:113]: Done

**Review by Madhura-08 — APPROVED** [review]

**Review by mariyappanp — APPROVED** [review]
Looks good.

**Pull request merged**